### PR TITLE
OpenAI working backend

### PR DESCRIPTION
**Pull request by UnMorrer**

### CODE REVIEW

File: workers/ai_worker/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+FROM python:3.11
+
+workdir /app
```
Comment (aPisC) [inline workers/ai_worker/Dockerfile]: WORKDIR

File: workers/ai_worker/__init__.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+from .orchestrator import Orchestrator
+import logging
+import sys
+
+logging.basicConfig(
```
Comment (aPisC) [inline workers/ai_worker/__init__.py]: not needed

File: workers/ai_worker/orchestrator.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
+from worker_base.orchestrator_base import OrchestratorBase
+from .modules.module_openai_trial import AIModule
+
+class Orchestrator(OrchestratorBase):
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, handler):
+        completion = AIModule(modelName="gpt-3.5-turbo")
+        retval = completion.execute(handler=handler)
```
Comment (aPisC) [inline workers/ai_worker/orchestrator.py:10]: return the response through handler

File: workers/console_app.py [hunk 4]
```diff
@@ -1,14 +1,5 @@
-import logging
-import sys
-
 from worker_base import ConsoleModelHandler
-
-logging.basicConfig(
-    level=logging.INFO,
-    format="[%(asctime)s] {%(filename)s:%(lineno)d} %(levelname)s - %(message)s",
```
Comment (aPisC) [inline workers/console_app.py:8]: revert it

**Review by aPisC — CHANGES_REQUESTED** [review]

File: workers/ai_worker/Dockerfile (hunk 1 above)
Comment (UnMorrer) [reply workers/ai_worker/Dockerfile]: DONE

File: workers/ai_worker/__init__.py (hunk 2 above)
Comment (UnMorrer) [reply workers/ai_worker/__init__.py]: REVERT

File: workers/ai_worker/orchestrator.py (hunk 3 above)
Comment (UnMorrer) [reply workers/ai_worker/orchestrator.py:10]: BASIC DONE

File: workers/console_app.py (hunk 4 above)
Comment (UnMorrer) [reply workers/console_app.py:8]: REVERT

**Review by aPisC — APPROVED** [review]

**Pull request merged**